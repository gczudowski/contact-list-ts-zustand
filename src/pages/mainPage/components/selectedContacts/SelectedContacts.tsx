import React, { ReactElement } from 'react';

interface Props {
  selectedContactsCount: number;
}

function SelectedContacts({ selectedContactsCount }: Props): ReactElement {
  return <div style={styles.selectedContactsContainer}>Selected contacts: {selectedContactsCount}</div>;
}

const styles = {
  selectedContactsContainer: {
    display: 'flex',
    justifyContent: 'center',
  },
};

export default SelectedContacts;
