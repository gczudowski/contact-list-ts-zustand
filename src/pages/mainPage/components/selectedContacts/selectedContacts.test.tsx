import React from 'react';
import renderer from 'react-test-renderer';
import SelectedContacts from './SelectedContacts';

describe('SelectedContacts', () => {
  const selectedContactsCountMock = 1;

  it('renders properly with initial props', () => {
    const component = renderer.create(<SelectedContacts selectedContactsCount={selectedContactsCountMock} />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
