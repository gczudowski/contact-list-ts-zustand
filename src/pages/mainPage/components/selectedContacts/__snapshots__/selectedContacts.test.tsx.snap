// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedContacts renders properly with initial props 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  Selected contacts: 
  1
</div>
`;
